lib/Doctrine/ORM/Tools/Pagination/Paginator.php

```diff
@@ -72,48 +66,39 @@ public function getQuery()
      *
      * @return bool Whether the query joins a collection.
      */
-    public function getFetchJoinCollection()
+    public function getFetchJoinCollection(): bool
     {
         return $this->fetchJoinCollection;
     }
 
     /**
      * Returns whether the paginator will use an output walker.
-     *
-     * @return bool|null
      */
-    public function getUseOutputWalkers()
+    public function getUseOutputWalkers(): ?bool
     {
         return $this->useOutputWalkers;
     }
 
     /**
      * Sets whether the paginator will use an output walker.
      *
-     * @param bool|null $useOutputWalkers
-     *
      * @return $this
      * @psalm-return static<T>
      */
-    public function setUseOutputWalkers($useOutputWalkers)
+    public function setUseOutputWalkers(?bool $useOutputWalkers): static
     {
         $this->useOutputWalkers = $useOutputWalkers;
 
         return $this;
     }
 
-    /**
-     * {@inheritdoc}
-     *
-     * @return int
-     */
     #[ReturnTypeWillChange]
```

Comment:
The attribute can be removed now.

Reply:
I have to look at the files one by one because for instance here, the constructor was not affected by Rector.